arrow/scripts/commons4gradle.sh

```diff
@@ -0,0 +1,190 @@
+#!/bin/bash
```

Comment:
What is this file doing? Is this change needed? cc\\ @rachelcarmena

Reply:
No it is not needed, I wanted to execute the `buildArrowDoc` gradle task and it couldn't find that file from my local instance. I will remove that. 

Still I couldn't execute the gradle task.  
Do you know how I can perform the Ank checks?

Reply:
There is a `buildArrowDoc` task available for me when checking out Arrow Fx 🤔 I think it relies on the other project being available locally, so `./gradlew gitPullAll`. Not 100% sure how that works. It's mostly easier to just run Ank on CI and fix all the issues that it raises 😅
That's why I made Ank work with `Validated` so long ago :joy: